packages/fiber/package.json

```diff
@@ -41,6 +41,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.13.10",
+    "@react-three/shared": ">=7.0.7",
```

Comment:
@joshuaellis Should @react-three/shared be a dependency or a peerDependency of fiber?